src/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /App
 RUN dotnet build App.csproj --configuration Release --output /app_output
 RUN dotnet publish App.csproj --configuration Release --output /app_output
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine AS final
+FROM mcr.microsoft.com/dotnet/aspnet:6.0.2-alpine3.14 AS final
```

Comment:
Same as for the build container, why are we pinning the alpine version?

Reply:
Changed to use latest.